specs/latest/2.0/webgl2.idl

```diff
@@ -10,19 +10,19 @@ typedef long long GLint64;
 typedef unsigned long long GLuint64;
 
 
-interface WebGLQuery : WebGLObject {
+[Exposed=(Window,Worker)] interface WebGLQuery : WebGLObject {
 };
 
-interface WebGLSampler : WebGLObject {
+[Exposed=(Window,Worker)] interface WebGLSampler : WebGLObject {
 };
 
-interface WebGLSync : WebGLObject {
+[Exposed=(Window,Worker)] interface WebGLSync : WebGLObject {
 };
 
-interface WebGLTransformFeedback : WebGLObject {
+[Exposed=(Window,Worker)] interface WebGLTransformFeedback : WebGLObject {
 };
 
-interface WebGLVertexArrayObject : WebGLObject {
+[Exposed=(Window,Worker)] interface WebGLVertexArrayObject : WebGLObject {
```

Comment:
It would be nice the `[Exposed...` bit was always on the line before `interface...`.

Reply:
Good point. Done throughout.